std/uni.d

```diff
@@ -6783,11 +6786,10 @@ public:
 
         See_Also: $(LREF Grapheme.valid)
     +/
-    ref opOpAssign(string op)(dchar ch)
+    ref opOpAssign(string op)(dchar ch) pure @nogc
```

Comment:
Are `pure` and `@nogc` inferred if you remove them from the signature? If yes, I we can remove them (I guess that they were useful to you when debugging the attributes on other code), if no, can you also add `nothrow`?